src/webui/webapplication.cpp

```diff
@@ -769,6 +771,33 @@ void WebApplication::action_command_bottomPrio()
     BitTorrent::Session::instance()->bottomTorrentsPriority(hashes);
 }
 
+void WebApplication::action_command_setLocation()
+{
+    CHECK_URI(0);
+    CHECK_PARAMETERS("hashes" << "location");
+
+    QStringList hashes = request().posts["hashes"].split("|");
+    QString newLocationPath = request().posts["location"].trimmed();
+
+    // check location exists
+    QFileInfo newLocation(newLocationPath);
+    if (!newLocation.exists()) {
+        qDebug() << "Location" << newLocationPath << "is invalid. Not moving files.";
+        return;
+    }
+
+    foreach (const QString &hash, hashes) {
+        BitTorrent::TorrentHandle *const torrent = BitTorrent::Session::instance()->findTorrent(hash);
+        if (torrent) {
+            // get old location
+            const QString oldLocation = torrent->savePath();
+            qDebug() << "Moving torrent" << torrent->hash() << "to" << newLocationPath;
```

Comment:
Currently qBittorrent displays a bunch of debug information, much of which is not only useless, but also hides the information that could be useful. When someone implements some code and/or tests it, he may need to use some debug messages (perhaps even redundant), but after that, most of these messages must be removed from the resulting code, because it is not required for constant use.